### PR TITLE
deleted redundant model files + ignored

### PR DESCRIPTION
**Pull request by EntranceJew**
[Redox said so, and I trust them](https://github.com/CFC-Servers/gm_addon_optimization_tricks/wiki/Optimizations-without-downsides)

### CODE REVIEW

**Review by TimGoll — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -58,3 +58,8 @@ addon.json
 .luarc.json
 
 ttt2.jpg
+
+# redundant source files
+*.dx80.vtx
+*.sw.vtx
+*.xbox.vtx
```
Comment (Histalek) [inline .gitignore:65]:
```suggestion

# redundant source files
*.dx80.vtx
*.sw.vtx
*.xbox.vtx
```

Please don't do this.

These files aren't expected to exist locally so they should not be ignored by git.

**Review by Histalek — CHANGES_REQUESTED** [review]
Looks good for the most part.

File: .gitignore (hunk 1 above)
Comment (EntranceJew) [reply .gitignore:65]: the files are generated by most model compilers, if someone else were to recompile or compile new files for this addon then the redundant files could potentially be reintroduced

Comment (Histalek) [reply .gitignore:65]: That is for reviewers to catch. 

With this it is way easier for the local directory content to diverge from the upstream repository content. Possibly leading to changes in behaviour.

**Pull request merged**